google/cloud/forseti/services/scanner/dao.py

```diff
@@ -183,6 +188,16 @@ def __repr__(self):
         return string.format(
             self.violation_type, self.resource_type, self.rule_name)
 
+    @staticmethod
+    def schema_update(table):
```

Comment:
Method name should start with verb first.  So, `update_schema()`, which is also consistent with `migrate_schema()`.  Or is there a reason to call this `schema_update()`?

Reply:
Renamed to update_schema